cirq/protocols/json.py

```diff
@@ -312,21 +344,39 @@ def to_json(obj: Any, file_or_fn, *, indent=2, cls=CirqEncoder):
             party classes, prefer adding the _json_dict_ magic method
             to your classes rather than overriding this default.
     """
-    if isinstance(file_or_fn, str):
+    if file_or_fn is None:
+        buffer = io.StringIO()
+        to_json(obj, buffer)
+        buffer.seek(0)
+        return buffer.read()
```

Comment:
If you want, you can replace this block with
```python
json.dumps(obj, indent=indent, cls=cls)
```

Reply:
Done.